800-63B copy-edit (+more changes for other docs)

rajdinh opened the pull request:
Complete copy-edit for 800-63B, plus small changes in other docs (typos, etc.).

800-63A will be ready by no later than Monday.

rajdinh committed:
Update references.md

rajdinh committed:
Update sec6_references.md

rajdinh committed:
Update references.md

rajdinh committed:
Update sec4_aal.md

rajdinh committed:
Update references.md

rajdinh committed:
Update references.md

rajdinh committed:
Update appA_memorized.md

rajdinh committed:
Update sec5_authenticators.md

rajdinh committed:
Update sec6_lifecycle.md

rajdinh committed:
Update sec10_usability.md

rajdinh committed:
Update sec5_authenticators.md

rajdinh committed:
Update sec5_authenticators.md

rajdinh committed:
Update sec6_lifecycle.md

rajdinh committed:
Update sec7_session.md

rajdinh committed:
Update sec4_aal.md

rajdinh committed:
Update sec9_privacy.md

rajdinh committed:
Update sec10_usability.md

rajdinh committed:
Update sec10_usability.md

rajdinh committed:
Update sec10_usability.md

rajdinh committed:
Update sec10_usability.md

rajdinh committed:
Update sec10_usability.md

rajdinh committed:
Update references.md

rajdinh committed:
Update sec4_aal.md

rajdinh committed:
Update sec5_authenticators.md

rajdinh committed:
Update sec7_session.md

rajdinh committed:
Update sec9_privacy.md

rajdinh committed:
Update sec10_usability.md

rajdinh committed:
Update sec10_usability.md

rajdinh committed:
Update sec4_aal.md

rajdinh committed:
Update sec1_2_introduction.md

rajdinh committed:
Update sec6_lifecycle.md

rajdinh committed:
Update sec6_lifecycle.md

rajdinh committed:
Update sec5_authenticators.md

rajdinh committed:
Update sec7_session.md

rajdinh committed:
Update sec1_2_introduction.md

rajdinh committed:
Update sec3_definitions.md

rajdinh committed:
Update sec7_session.md

rajdinh committed:
Update sec9_privacy.md

rajdinh committed:
Update sec8_security.md

rajdinh committed:
Update sec7_session.md

rajdinh committed:
Update sec6_lifecycle.md

rajdinh committed:
Update sec10_usability.md

rajdinh committed:
Update sec10_usability.md

rajdinh committed:
Update sec10_usability.md

rajdinh committed:
Update sec10_usability.md

rajdinh committed:
Update sec10_usability.md

rajdinh committed:
Update sec10_usability.md

rajdinh committed:
Update sec10_usability.md

rajdinh committed:
Update sec10_usability.md

rajdinh committed:
Update sec10_usability.md

rajdinh committed:
Update sec3_definitions.md

rajdinh committed:
Update sec4_aal.md

rajdinh committed:
Update sec5_authenticators.md

rajdinh committed:
Update sec6_lifecycle.md

rajdinh committed:
Update sec7_session.md

rajdinh committed:
Update sec8_security.md

rajdinh committed:
Update sec10_usability.md

rajdinh committed:
Update sec4_aal.md

rajdinh committed:
Update sec9_privacy.md

rajdinh committed:
Update sec9_privacy.md

rajdinh committed:
Update sec4_aal.md

rajdinh committed:
Update sec3_definitions.md

rajdinh committed:
Update sec1_2_introduction.md

rajdinh committed:
Update sec3_definitions.md

rajdinh committed:
Update sec4_aal.md

rajdinh committed:
Update sec9_privacy.md

rajdinh committed:
Update sec1_2_introduction.md

rajdinh committed:
Update sec10_usability.md

rajdinh committed:
Update sec4_aal.md

rajdinh committed:
Update sec10_usability.md

rajdinh committed:
Update sec5_authenticators.md

rajdinh committed:
Update sec3_definitions.md

abdinh committed:
Merge remote-tracking branch 'refs/remotes/usnistgov/nist-pages' into nist-pages

rajdinh committed:
Update sec8_security.md

rajdinh committed:
Update sec3_definitions.md

rajdinh committed:
Update sec7_session.md

rajdinh committed:
Update sec8_security.md

rajdinh committed:
Update sec5_authenticators.md

rajdinh committed:
Update sec5_authenticators.md

rajdinh committed:
Update sec5_authenticators.md

rajdinh committed:
Update sec3_definitions.md

rajdinh committed:
Update sec4_aal.md

rajdinh committed:
Update sec5_authenticators.md

rajdinh committed:
Update sec3_definitions.md

rajdinh committed:
Update sec3_definitions.md

rajdinh committed:
Update sec3_definitions.md

rajdinh committed:
Update appA_memorized.md

rajdinh committed:
Update sec9_privacy.md

rajdinh committed:
Update sec4_aal.md

rajdinh committed:
Update sec5_authenticators.md

rajdinh committed:
Update sec6_lifecycle.md

rajdinh committed:
Update sec8_security.md

rajdinh committed:
Update sec10_usability.md

rajdinh committed:
Update sec5_authenticators.md

rajdinh committed:
Update sec3_definitions.md

rajdinh committed:
Update sec4_aal.md

rajdinh committed:
Update sec10_usability.md

rajdinh committed:
Update sec5_authenticators.md

rajdinh committed:
Update sec5_authenticators.md

rajdinh committed:
Update sec10_usability.md

rajdinh committed:
Update sec5_authenticators.md

rajdinh committed:
Update cover.md

rajdinh committed:
Update sec1_2_introduction.md

rajdinh committed:
Update sec3_definitions.md

rajdinh committed:
Update sec4_model.md

rajdinh committed:
Update sec5_xAL.md

abdinh committed:
Merge remote-tracking branch 'refs/remotes/usnistgov/nist-pages' into nist-pages

abdinh committed:
Merge branch 'nist-pages' of https://github.com/rajdinh/800-63-3 into nist-pages

rajdinh committed:
Update sec1_2_introduction.md

rajdinh committed:
Update sec3_definitions.md

rajdinh committed:
Update sec5_authenticators.md

rajdinh committed:
Update sec10_usability.md

rajdinh committed:
Update appA_memorized.md

rajdinh committed:
Update sec1_2_introduction.md

rajdinh committed:
Update sec4_federation.md

rajdinh committed:
Update sec8_security.md

rajdinh committed:
Update sec9_privacy.md

rajdinh committed:
Update sec10_usability.md

rajdinh committed:
Update sec11_examples.md

rajdinh committed:
Update sec1_2_introduction.md

rajdinh committed:
Update sec4_ial.md

rajdinh committed:
Update sec5_proofing.md

rajdinh committed:
Update sec6_dc.md

rajdinh committed:
Update sec10_usability.md

jimfenton commented on sp800-63a/sec4_ial.md:
```diff
@@ -72,22 +72,22 @@ The following requirements apply to any CSP performing identity proofing at IAL2
 7. The identity proofing and enrollment processes SHALL be performed according to an applicable written policy or *practice statement* that specifies the particular steps taken to verify identities.
-3. The CSP SHALL maintain a record of all steps taken to verify the identity of the applicant and SHALL record the types of identity evidence presented in the proofing process. The CSP SHALL conduct a privacy risk assessment to determine:
+8. The CSP SHALL maintain a record of all steps taken to verify the identity of the applicant and SHALL record the types of identity evidence presented in the proofing process. The CSP SHALL conduct a privacy risk assessment to determine:
 
```
Spacing is inconsistent between items in this numbered list. Some items are double-spaced, others not.

jimfenton commented on sp800-63a/sec6_dc.md:
```diff
@@ -32,12 +32,12 @@ The following requirements detail how a CSP determines the existence of a enroll
 
 
-## <a name="dc-ial3"></a>6.4. AAL 3 Requirements
+## <a name="dc-ial3"></a>6.3. AAL 3 Requirements
```
I don't know if this label is used anywhere, but it would be more logical for it to be named "dc-aal3".

jimfenton commented on sp800-63b/sec10_usability.md:
```diff
@@ -10,27 +10,24 @@ _This section is informative._
-In this section, the term "users" means "claimants" or "subscribers."
-
-Guidelines and considerations are described from the users' perspective.
+"Guidelines and considerations are described from the users' perspective.
```
Leading quote is misplaced (should close subscribers in preceding paragraph).

jimfenton commented on sp800-63b/sec10_usability.md:
```diff
@@ -41,210 +38,218 @@ Usability considerations that are applicable to most authenticators are describe
 
 Usability considerations for typical usage of all authenticators include:
```
If changing this from a bulleted list to a number of paragraphs, need to introduce this differently.

jimfenton commented on sp800-63b/sec1_2_introduction.md:
```diff
@@ -4,29 +4,29 @@
 The ongoing authentication of subscribers is central to this process. Subscriber authentication is performed by verifying that the claimant controls one or more *authenticators* (called *tokens* in earlier editions of SP 800-63) associated with a given subscriber. A successful authentication results in the assertion of an identifier, either pseudonymous or non-pseudonymous, and optionally other i ...
 
-This document provides guidelines on types of authentication processes, including choices of authenticators, that may be used at various *Authenticator Assurance Levels* (AALs). It also provides guidelines on the lifecycle of authenticators, including revocation in the event of loss or theft.
+This document provides recommendations on types of authentication processes, including choices of authenticators, which may be used at various *Authenticator Assurance Levels* (AALs). It also provides recommendations on the lifecycle of authenticators, including revocation in the event of loss or theft.
```
I believe "that" to be correct because it is introducing a restrictive clause. http://blog.oxforddictionaries.com/2012/09/that-who-which/

jimfenton commented on sp800-63b/sec3_definitions.md:
```diff
@@ -9,7 +9,7 @@ There is a variety of terms used in the area of authentication. While the defini
 
 #### Active Attack
-An attack on the authentication protocol where the attacker transmits data to the claimant, Credential Service Provider, verifier, or Relying Party. Examples of active attacks include man-in-the-middle, impersonation, and session hijacking.
+An attack on the authentication protocol where the attacker transmits data to the claimant, Credential Service Provider (CSP), verifier, or RP. Examples of active attacks include man-in-the-middle (MitM), impersonation, and session hijacking.
```
I don't understand why this spells out Credential Service Provider but not Relying Party. People don't read definitions linearly so I would like to require as little context as possible in the definitions.

rajdinh replied:
The general pattern I discussed with Paul that I'm following is that if the acronym was introduced in an earlier section (in this case the intro for RP) we go with the acronym from that point forward in the rest of document sections. I don't have a problem with spelling out RP here though if it will help the definition stand on its own.

jimfenton committed:
Correct residual spacing of xALn in -3, -63A, and -63B.

jimfenton reviewed (commented):
Initial comments; this is a large commit so will review in sections.

rajdinh committed:
Update sec4_ial.md

rajdinh committed:
Update sec4_ial.md

rajdinh committed:
Update sec1_2_introduction.md

rajdinh committed:
Update sec10_usability.md

rajdinh committed:
Update sec3_definitions.md

rajdinh committed:
Minor reference section changes

- Fixed header sizing (headers throughout 63A do not match size of headers in other docs)
- Added period to end of A-130 reference
- Removed highlighting of ISO 9241-11 reference
- Added "this is informative statement"

rajdinh committed:
800-63A Section 1 and 2 changes

- Corrected size of headers
- Increased use of acronyms (CSP, IAL, LOA)
- Minor changes (hyphen for CSP-issued, hyphens instead of colons for listing IALs and describing them)
- Added informative/normative table for the doc in Section 2

rajdinh committed:
Minor punctuation change for 63C table headers

Changed colons to periods in table headers for consistency across docs.

rajdinh committed:
Minor punctuation change for table header in 63C

Replaced colon with period for table header in Section 9.

rajdinh committed:
Minor punctuation change for 63C figure subtitles

Changed colons in figure subtitles to periods (Section 4).

rajdinh committed:
Minor 63C punctuation changes for figure subtitles 

- Changed colons to periods in figure subtitles
- Capitalization for Figure 6-2 subtitle

jimfenton commented on sp800-63b/sec5_authenticators.md:
```diff
@@ -4,26 +4,26 @@
   </div>
 
-##### 5.1.1.1. Memorized Secret Authenticators
+#### 5.1.1.1. Memorized Secret Authenticators
```
Since 5.1.1 is a #### header, shouldn't 5.1.1.1 be ##### ?

rajdinh replied:
That's correct. However, I intentionally left it as #### because beyond that the headers get very small and hard to discern from the other text. Capping it at 4 #'s is a similar approach to what we did for NISTIR 8112.

jimfenton commented on sp800-63b/sec5_authenticators.md:
```diff
@@ -4,26 +4,26 @@
 Memorized secrets SHALL be at least 8 characters in length if chosen by the subscriber; memorized secrets chosen randomly by the CSP or verifier SHALL be at least 6 characters in length and MAY be entirely numeric.  Since the CSP or verifier may disallow some choices of memorized secrets based on their appearance on a blacklist of compromised values, the subscriber SHALL choose a different memori ...
 
-##### 5.1.1.2. Memorized Secret Verifiers
+#### 5.1.1.2. Memorized Secret Verifiers
```
Same comment.

rajdinh replied:
Same response as above.

jimfenton commented on sp800-63b/sec5_authenticators.md:
```diff
@@ -63,28 +63,28 @@ Verifiers SHALL store memorized secrets in a form that is resistant to offline a
   </div>
 
-##### 5.1.2.1 Look-up Secret Authenticators
+#### 5.1.2.1. Look-up Secret Authenticators
```
Same comment as 5.1.1.1

rajdinh replied:
Same response as above.

jimfenton commented on sp800-63b/sec5_authenticators.md:
```diff
@@ -63,28 +63,28 @@ Verifiers SHALL store memorized secrets in a form that is resistant to offline a
 If the authenticator uses look-up secrets sequentially from a list, the subscriber MAY dispose of used secrets, but only after a successful authentication.
 
-##### 5.1.2.2. Look-up Secret Verifiers
+#### 5.1.2.2. Look-up Secret Verifiers
```
Same comment as 5.1.1.1

rajdinh replied:
Same response as above.

jimfenton commented on sp800-63b/sec5_authenticators.md:
```diff
@@ -97,7 +97,7 @@ The purpose of the secret is to securely bind the authentication operation on th
   </div>
 
-##### 5.1.3.1. Out-of-Band Authenticators
+#### 5.1.3.1. Out-of-Band Authenticators
```
Same comment as 5.1.1.1

rajdinh replied:
Same response as above.

jimfenton commented on sp800-63b/sec5_authenticators.md:
```diff
@@ -117,7 +117,7 @@ If the out-of-band authenticator sends an approval message over the secondary co
 * The authenticator SHALL present a secret received via the secondary channel from the verifier and prompt the claimant to verify the consistency of that secret with the primary channel, prior to accepting a yes/no response from the claimant. It SHALL then send that response to the verifier.
 
-##### 5.1.3.2. Out-of-Band Verifiers
+#### 5.1.3.2. Out-of-Band Verifiers
```
Same comment as 5.1.1.1. And so on...(won't keep pointing these out)

rajdinh replied:
Same response as above.

jimfenton commented on sp800-63b/sec5_authenticators.md:
```diff
@@ -317,15 +317,15 @@ CSPs SHALL provide subscriber instructions on how to appropriately  protect the
 Additional techniques MAY be used to prioritize authentication attempts that are likely to come from the subscriber over those that are more likely to come from an attacker:
 
-- Requiring the claimant to complete a Completely Automated Public Turing test to tell Computers and Humans Apart (CAPTCHA) before attempting authentication
+- Requiring the claimant to complete a Completely Automated Public Turing test to tell CAPTCHA before attempting authentication.
```
New wording doesn't make sense. The old wording provided the meaning of the abbreviation CAPTCHA, and the new one doesn't. Capitalization of the words was based on those included in the abbreviation.

jimfenton commented on sp800-63b/sec5_authenticators.md:
```diff
@@ -335,7 +335,7 @@ When the subscriber successfully authenticates, the verifier SHOULD disregard an
 For a variety of reasons, this document supports only limited use of biometrics for authentication. These include:
 
-- Biometric False Match Rates (FMR) and False Non-Match Rates (FNMR) do not provide confidence in the authentication of the subscriber by themselves. In addition, FMR and FNMR do not account for spoofing attacks.
+- FMR and FNMR do not provide confidence in the authentication of the subscriber by themselves. In addition, FMR and FNMR do not account for spoofing attacks.
```
The abbreviations FMR and FNMR are only introduced in the definition of EER in Section 3. Please leave the words.

jimfenton commented on sp800-63b/sec6_lifecycle.md:
```diff
@@ -18,46 +18,52 @@ The record created by the CSP SHALL contain the date and time the authenticator
-At IAL 2 and above, identifying information is associated with the online identity and the subscriber has undergone an identity proofing process as described in SP 800-63A. As a result, authenticators at the same AAL as the desired IAL SHALL be bound to the account. For example, if the subscriber has successfully completed proofing at IAL 2, AAL 2 or 3 authenticators are appropriate to bind to th ...
+At IAL2 and above, identifying information is associated with the online identity and the subscriber has undergone an identity proofing process as described in SP 800-63A. As a result, authenticators at the same AAL as the desired IAL SHALL be bound to the account. For example, if the subscriber has successfully completed proofing at IAL2, AAL2 or AAL3 authenticators are appropriate to bind to th ...
 
 Enrollment and binding MAY be broken up into a number of separate physical encounters or electronic transactions. Two electronic transactions are considered to be separate if they are not part of the same protected session. In these cases, the following methods SHALL be used to ensure that the same party acts as applicant throughout the processes:
```
By eliminating one level of the bulleted list, it's less clear where the "following methods" end and the following text begins. I prefer to keep the previous bulleted list structure, but at the least we need to make it clear which things comprise the methods we are requiring.

rajdinh replied:
Should now be corrected into a structure similar to the previous list, but using a, b, and c at the 2nd level instead of numbers again.

jimfenton commented on sp800-63b/sec6_lifecycle.md:
```diff
@@ -18,46 +18,52 @@ The record created by the CSP SHALL contain the date and time the authenticator
+
+#### <a name="post-enroll-bind"></a> 6.1.2. Post-Enrollment Binding
+
+#### 6.1.2.1. Binding of Additional Authenticator at Existing AAL
```
Same comment as in Section 5 on fourth-level header styles.

rajdinh replied:
Same response as above as far as capping it at ####.

jimfenton commented on sp800-63b/sec9_privacy.md:
```diff
@@ -6,26 +6,26 @@
-[Section 4.4](#aal_privacy) does not permit the CSP to use information about authenticators that is collected and maintained in the authentication process for any purpose other than authentication or to comply with law or legal process, unless the CSP provides clear notice and obtains consent from the subscriber for additional uses. Care should be taken to ensure that use of such information is l ...
+[Section 4.4](#aal_privacy) does not permit the CSP to use information about authenticators that is collected and maintained in the authentication process for any purpose other than authentication or to comply with law or legal process, unless the CSP provides clear notice and obtains consent from the subscriber for additional uses. Care should be taken to ensure that use of such information is l ...
 
 ### 9.4. Agency Specific Privacy Compliance 
```
Maybe "Agency-specific" ? (I'm not sure)

jimfenton commented on sp800-63b/sec1_2_introduction.md:
```diff
@@ -4,29 +4,29 @@
 _This section is informative._
 
-This document and its companion documents, [SP 800-63-3](sp800-63-3.html), [SP 800-63A](sp800-63a.html), and [SP 800-63C](sp800-63c.html), provide technical guidelines to agencies for the implementation of digital authentication.
+This document and its companion documents, [Special Publication (SP) 800-63-3](sp800-63-3.html), [SP 800-63A](sp800-63a.html), and [SP 800-63C](sp800-63c.html), provide technical guidelines to agencies for the implementation of digital authentication.
```
Later sections use double brackets when linking to other Special Publications. I'm not sure whether we want to show the brackets to make it clear that they're references like that, but in any case we should be consistent.

rajdinh replied:
I will add double brackets to links for other SPs - you'll see commits to change sections in -3, 63B, and 63C, and I will include any changes for 63A in the next pull request as well.

jimfenton commented on sp800-63c/sec4_federation.md:
```diff
@@ -90,7 +90,7 @@ The subscriber SHALL receive explicit notice and be able to provide positive con
 ### <a name="privacy-reqs"></a> 4.2. Privacy Requirements
 
-Federation involves the transfer of personal attributes from a third party, the IdP, that is not otherwise involved in a transaction. Federation also potentially gives the IdP broad visibility into subscriber activities. Accordingly, there are specific privacy requirements associated with federation. 
+Federation involves the transfer of personal attributes from a third party, the IdP, which is not otherwise involved in a transaction. Federation also potentially gives the IdP broad visibility into subscriber activities. Accordingly, there are specific privacy requirements associated with federation. 
```
Should be "that" because it introduces a restrictive clause.

jimfenton commented on sp800-63b/sec10_usability.md:
```diff
@@ -39,212 +36,218 @@ Positive user authentication experiences are integral to the success of an organ
 **_Intermittent Events_**
 
-Usability considerations for intermittent events include:
+Usability considerations for intermittent events include when users create and change memorized secrets:  
```
This rewording doesn't make sense.  Maybe something like:

The use and changing of memorized secrets are intermittent events. Usability considerations for these events include:

jimfenton reviewed (changes requested):
I'm getting on a plane, so rather than pushing corrections to your branch as we discussed, I will leave these for you to correct if you're able to. If you're not able to get to these today, I can push corrections tomorrow.

rajdinh committed:
Update sec5_authenticators.md

rajdinh committed:
Update sec5_authenticators.md

rajdinh committed:
Update sec9_privacy.md

rajdinh committed:
Update sec10_usability.md

rajdinh committed:
Update sec4_federation.md

rajdinh committed:
Update sec1_2_introduction.md

rajdinh committed:
Update sec4_model.md

rajdinh committed:
Update sec5_xAL.md

rajdinh committed:
Double brackets for SP links

rajdinh committed:
Double brackets for SP links

rajdinh committed:
Double brackets for SP links

rajdinh committed:
Double brackets for SP links

rajdinh committed:
Double brackets for SP links

rajdinh committed:
Double brackets for SP links

rajdinh committed:
Removed redundant references to 63C within 63C

rajdinh committed:
Broke remote/physical list back into previous list structure

Now reads 1. Remote and 2. Physical with a, b, and c sub-bullets.

rajdinh committed:
Reverting back to previous format for 63B Usability

jimfenton committed:
Definition updates: MF -> MFA, IdP

jimfenton committed:
Clean up reference call-outs.

jimfenton committed:
Straighten quotes in Appendix A
